network-config-analyzer/CLExecute.py

```diff
@@ -22,73 +22,30 @@ def __init__(self, base_ns_list=None, base_pod_list=None, ns_list=None, pod_list
         self.output_config = OutputConfiguration({'outputFormat': output_format, 'outputPath': output_path,
                                                   'prURL': pr_url})
 
-    def sanity(self, np_list):
-        """
-        Runs a set of sanity check on a given list of network policies
-        :param str np_list: A set of policies
-        :return: 0 if all sanity checks passed. 1 otherwise
-        :rtype: int
-        """
-        network_config = NetworkConfig(np_list, self.peer_container, [np_list])
-        query_output = '\n'
-        sanity_res = NetworkConfigQuery.SanityQuery(network_config).exec()
-        query_output += sanity_res.output_result
-        if not sanity_res.bool_result:
-            query_output += sanity_res.output_explanation
-        query_output += '\n'
-        self.output_config.print_query_output(query_output)
-        return int(not sanity_res.bool_result)
+    def execute_single_config_query(self, query_name, np1_list):
+        network_config1 = NetworkConfig(np1_list, self.peer_container, [np1_list])
+        return NetworkConfigQueryRunner(query_name, [network_config1], self.output_config).run_query()
 
-    def equivalence(self, np1_list_location, np2_list_location):
+    def execute_pair_configs_query(self, query_name, np1_list_location, np2_list_location):
         """
         Runs an equivalence-checking query between two sets of policies
+        :param str query_name: the name of the arg.query
         :param str np1_list_location: First set of policies
-        :param str np2_list_location: Second set of policies
-        :return: 0 if the sets of policies are semantically equivalent. 1 otherwise
+        :param str np2_list_location: Second set of policies (or the base_policies_list)
+        :return: result of executing the query
         :rtype: int
         """
         network_config1 = NetworkConfig(np1_list_location, self.peer_container, [np1_list_location])
         network_config2 = NetworkConfig(np2_list_location, self.peer_container, [np2_list_location])
```

Comment:
I think there is an issue here to assign for both network configs the same peer_container.
for the case of semantic diff query , it was originally as follows (on master):
```
network_config1 = NetworkConfig(np1_list_location, self.base_peer_container, [np1_list_location])
network_config2 = NetworkConfig(np2_list_location, self.peer_container, [np2_list_location])
```

so should refer to both ` self.base_peer_container` and ` self.peer_container`

Reply:
Hi @adisos 
I fixed the above failing output.
In the new commit all failing output tests , are failing because of whitespaces and empty lines.
except for the cmd_line tests that include connectivity map query, I added the "config name" to the output, in order to be consistent with the output of the scheme files' queries' output (didn't affect the results).
so if this is OK for you, would like to know how to override the expected outputs safely in order to commit them too

Reply:
hi @shireenf-ibm , it looks ok now, you can override expected output by running the following: 
`tests/run_all_tests.py --type=output --action=override_expected_output`